pkg/mcs/tso/server/server.go

```diff
@@ -68,7 +68,7 @@ const (
 	pdRootPath = "/pd"
 	// tsoPrimaryPrefix defines the key prefix for keyspace group primary election.
 	// The entire key is in the format of "/ms/<cluster-id>/tso/<group-id>/primary" in which
-	// <group-id> is 5 digits integer with leading zeros. For now we use 0 as the default cluster id.
+	// <group-id> is 5 digits integer with leading zeros. 0 is the default cluster id.
```

Comment:
Maybe we don't need the default cluster ID?

Reply:
you're right. use the cluster id generated instead.